cropduster/models.py

```diff
@@ -427,15 +432,21 @@ def save_size(self, size, thumb=None, image=None, tmp=False, standalone=False,
         if not image and not self.image:
             raise Exception("Cannot save sizes without an image")
 
-        image = image or PIL.Image.open(safe_str_path(self.image.path))
+        if image:
```

Comment:
Could we modify this conditional to `if not image` and execute what we currently have in the else block?

Reply:
Ah, yes, most certainly 👍